packages/api-explorer/src/ApiExplorer.tsx

```diff
@@ -106,15 +106,18 @@ const ApiExplorer: FC<ApiExplorerProps> = ({
 
   useEffect(() => {
     const loadSpec = async () => {
+      let api
```

Comment:
This useEffect can be simplified to (get rid of the .then(...))
```
  useEffect(() => {
    const loadSpec = async () => {
      if (!spec.api) {
        const newSpec = { ...spec }
        const api = await loadSpecApi(newSpec)
        if (api) {
          specDispatch(updateSpecApi(spec.key, api))
        }
      }
    }
    loadSpec()
  }, [spec])
```

Reply:
funny. That's what was there originally but the linter kept complaining about the non-async call.

Reply:
restored